fix(astro): display diagnostic advices with the correct location (#10329)
biomejs/biome: .changeset/bright-dragons-clean.md

---
"@biomejs/biome": patch
---

> Fixed diagnostics' advices in Astro files that displayed wrong locations.

Reviewer: ```suggestion
Fixed an issue where diagnostics showed an incorrect location in Astro files. 
```
